### DISCUSSION TITLE
Can the message subscription be pre-created?

### DISCUSSION THREAD

**Discussion by zhuanxuhit** [milvus-io/milvus#26014] [Q&A and General discussion]
querynode 新增订阅的时候，能否是预先就建立的？

**Comment by yhmo** [comment]
在v2.2.x版本上如果预先建立是不行的。最新的v2.3版本(非beta版)才能支持预先建立pulsar/kafka的topic，正式版本的2.3预计在最近一两周左右发布。在2.3中，将会看到在配置文件milvus.yaml里多了这些选项:
![20230731-153615](https://github.com/milvus-io/milvus/assets/2282099/4a6c57ff-ae1e-4f5c-b3e6-00ae12c5fd4b)

**Reply by zhuanxuhit** [reply]
This only specifies a pre-created topic, can the subscription also be specified as pre-created? @yhmo

**Reply by yhmo** [reply]
> This only specifies a pre-created topic, can the subscription also be specified as pre-created? @yhmo

Not possible

**Comment by yhmo** [comment]
另外：在milvus社区里请尽量使用英文提问和交流，这样对各地区不同国家的用户比较友好。
